### PR TITLE
feat(base): add cancelOrders emulation

### PR DESCRIPTION
**Pull request by pcriadoperez**
Add cancelOrders to base Exchange and emulate.

### CODE REVIEW

**Comment by pcriadoperez** [comment]
```python
binanceusdm.cancelOrders([3377507576],BTC/USDT:USDT)
[{'amount': 0.001,
  'average': None,
  'clientOrderId': 'JynjHn8e0PAX01gZ5E3TCp',
  'cost': 0.0,
  'datetime': '2023-06-18T15:51:22.970Z',
  'fee': {'cost': None, 'currency': None, 'rate': None},
  'fees': [{'cost': None, 'currency': None, 'rate': None}],
  'filled': 0.0,
  'id': '3377507576',
  'info': {'avgPrice': '0.00',
           'clientOrderId': 'JynjHn8e0PAX01gZ5E3TCp',
           'closePosition': False,
           'cumQty': '0.000',
           'cumQuote': '0.00000',
           'executedQty': '0.000',
           'orderId': '3377507576',
           'origQty': '0.001',
           'origType': 'LIMIT',
           'positionSide': 'BOTH',
           'price': '10000.00',
           'priceProtect': False,
           'reduceOnly': False,
           'side': 'BUY',
           'status': 'CANCELED',
           'stopPrice': '0.00',
           'symbol': 'BTCUSDT',
           'timeInForce': 'GTC',
           'type': 'LIMIT',
           'updateTime': '1687103482970',
           'workingType': 'CONTRACT_PRICE'},
  'lastTradeTimestamp': None,
  'lastUpdateTimestamp': 1687103482970,
  'postOnly': False,
  'price': 10000.0,
  'reduceOnly': False,
  'remaining': 0.001,
  'side': 'buy',
  'status': 'canceled',
  'stopPrice': None,
  'symbol': 'BTC/USDT:USDT',
  'timeInForce': 'GTC',
  'timestamp': 1687103482970,
  'trades': [],
  'triggerPrice': None,
  'type': 'limit'}]```

File: ts/src/base/Exchange.ts [hunk 1]
```diff
@@ -3322,6 +3322,19 @@ export default class Exchange {
         throw new NotSupported (this.id + ' cancelOrder() is not supported yet');
     }
 
+    async cancelOrders (ids: string[], symbol: string = undefined, params = {}): Promise<any> {
```
Comment (carlosmiei) [inline ts/src/base/Exchange.ts:3325]: @pcriadoperez Usually even when we emulate a unified method we still perform a single REST call which is not the case here so I'm not sure yet if we should open this precedent or not, for instance should we also emulate `fetchOrders` with multiple `fetchOrder` calls?

@kroitor what do you think?

**Comment by pcriadoperez** [comment]
True, we do have that as a general rule. I wrote it that way because I saw
it similar in editOrder, that cancels, and then creates an orders.

**Comment by carlosmiei** [comment]
> 

@pcriadoperez I understand but even there, the amount of calls is fixed

File: ts/src/base/Exchange.ts (hunk 1 above)
Comment (kroitor) [reply ts/src/base/Exchange.ts:3325]: I'd rather not emulate the missing functionality (cause that's a lot of emulation to add across the lib). I would probably leave the rate limiting control one the user side for cases like these.

**Pull request closed without merging**